src/date-picker/README.md

```diff
@@ -36,8 +36,7 @@ You have also ability to disable the datePicker
 | Name   |      Type      |  Description |
 |:----------|:-------------:|:------|
 | `todayBottonLabel` | string | Creates today button with custom name, which selects today date from calendar, and fill the datePicker input. |
-| `cancelButtonLabel` | string | Creates cancel button with custom name,
-which closes the calendar. |
+| `cancelButtonLabel` | string | Creates cancel button with custom name, which closes the calendar. |
```

Comment:
Switch the wording on `cancelButtonLabel` and `todayButtonLabel` from "Creates" to "Renders"